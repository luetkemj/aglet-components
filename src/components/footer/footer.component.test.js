import React from 'react';
import renderer from 'react-test-renderer';
import Footer from './footer.component';

test('Footer renders correctly', () => {
  const tree = renderer
    .create(<Footer repo="https://github.com/luetkemj/aglet-components" />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
