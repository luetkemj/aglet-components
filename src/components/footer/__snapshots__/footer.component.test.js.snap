// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="actions"
  >
    <div
      className="left"
    >
      <p>
        <a
          href="http://luetkemj.github.io/"
        >
          Made with 
          <span
            className="heart"
          >
            â™¥
          </span>
           in Austin, Texas
        </a>
      </p>
    </div>
    <div
      className="right"
    >
      <p
        className="bug"
      >
        <a
          href="https://github.com/luetkemj/aglet-componentsissues"
        >
          Report a bug
        </a>
      </p>
      <p
        className="feature"
      >
        <a
          href="https://github.com/luetkemj/aglet-componentsissues"
        >
          Request a feature
        </a>
      </p>
    </div>
  </div>
</div>
`;
